nr/data/.flows.json.backup

```diff
@@ -0,0 +1 @@
+[{"id":"ee1936cd.b6d21","type":"tab","label":"Flow 1","disabled":false,"info":""},{"id":"85d0c327.d3e95","type":"mqtt-broker","z":"","name":"MQTT","broker":"mqtt","port":"8883","tls":"62b8b95c.26b7d8","clientid":"nr.scifi.farm","usetls":true,"compatmode":false,"keepalive":"60","cleansession":true,"willTopic":"","willQos":"0","willPayload":"","birthTopic":"","birthQos":"0","birthPayload":""},{"id":"62b8b95c.26b7d8","type":"tls-config","z":"","name":"mqtt","cert":"/run/secrets/nr_cert","key":"/run/secrets/nr_key","ca":"/run/secrets/ca","certname":"","keyname":"","caname":"","verifyservercert":true},{"id":"522474fa.f9580c","type":"mqtt in","z":"ee1936cd.b6d21","name":"","topic":"planteyes/test","qos":"2","broker":"85d0c327.d3e95","x":460,"y":240,"wires":[["cce36914.a60c5"]]},{"id":"cce36914.a60c5","type":"mqtt out","z":"ee1936cd.b6d21","name":"","topic":"planteyes/success","qos":"","retain":"","broker":"85d0c327.d3e95","x":1000,"y":280,"wires":[]}]
```

Comment:
Is this like, default test data for making sure the MQ stuff works?

Reply:
Yeah, basically. It's not really test data though, it's just a flow with a MQTT in and out node. If it connects, it'll show "connected" next to the node in NodeRED. More than that, this is just a backup of .flows.json, which is actually what's getting used.

I don't think it's necessary, so I've removed this file and added *.backup to .gitignore.